### PR TITLE
Add Ctrl+F search shortcut for ACE Arsenal

### PR DESCRIPTION
**Pull request by alganthe**
**When merged this pull request will:**
- Add Ctrl+F, Ctrl+A, and Ctrl+X support for ACE Arsenal

Pressing it while no search field are in focus: select left field
Pressing it while a search field is in focus: select the other field.

### CODE REVIEW

**Comment by commy2** [comment]
This is very nice!

File: addons/arsenal/functions/fnc_onKeyDown.sqf [hunk 1]
```diff
@@ -99,6 +103,14 @@ if (!GVAR(leftSearchbarFocus) && {!GVAR(rightSearchbarFocus)}) then {
         case (_keyPressed == DIK_V && {_ctrlState}): {
             _return = false;
         };
```
Comment (PabstMirror) [inline addons/arsenal/functions/fnc_onKeyDown.sqf]: Maybe ignore ctrl+x as well?
```
        case (_keyPressed == DIK_X && {_ctrlState}): {
            _return = false;
        };
```

Comment (alganthe) [reply addons/arsenal/functions/fnc_onKeyDown.sqf]: You mean allow cut and paste?

**Comment by alganthe** [comment]
Added Ctrl+A and Ctrl+X support as requested.

**Pull request merged**